database/src/main/scala/cromwell/database/slick/tables/CallCachingResultSimpletonComponent.scala

```diff
@@ -0,0 +1,38 @@
+package cromwell.database.slick.tables
+
+import cromwell.database.sql.tables.CallCachingResultSimpletonEntry
+
+trait CallCachingResultSimpletonComponent {
+
+  this: DriverComponent with CallCachingMetainfoComponent =>
+
+  import driver.api._
+
+  class CallCachingResultSimpletonEntries(tag: Tag) extends Table[CallCachingResultSimpletonEntry](tag, "JOB_STORE") {
```

Comment:
I was struggling to think of a good name for a single WDL element with a simple type 😕